bin/cdxgen.js

```diff
@@ -419,6 +423,13 @@ if (process.argv[1].includes("cbom")) {
   options.specVersion = 1.6;
   options.deep = true;
 }
+if (process.argv[1].includes("cdxgen-secure")) {
+  console.log(
+    "NOTE: Secure mode only restricts cdxgen from performing certain activities such as package installation. It does not provide security guarantees in the presence of malicious code.",
```

Comment:
Wondering if we this message is enough. Should we use a different word instead of secure?